tvb_framework/tvb/adapters/creators/siibra_creator.py

```diff
@@ -57,23 +57,32 @@ def init_siibra_options():
     # has data and corresponds with the current API of siibra
     parcellations = [siibra_base.DEFAULT_PARCELLATION]
 
+    # get available cohorts
+    cohorts = siibra_base.get_cohorts_for_sc(parcellations[0])
+    print(cohorts)
+
     atlas_dict = {}
     parcellation_dict = {}
+    cohort_dict = {}
 
     for a_name in atlases:
         atlas_dict[a_name] = a_name
 
     for p_name in parcellations:
         parcellation_dict[p_name] = p_name
 
+    for c_name in cohorts:
+        cohort_dict[c_name.upper()] = c_name.upper()
```

Comment:
use dictionary comprehension instead `cohort_dict = {(y := c_name.upper()) : y for c_name in cohorts}`